server/container/mailbox-adapter/src/test/java/org/apache/james/adapter/mailbox/MailboxManagementTest.java

```diff
@@ -147,11 +149,14 @@ public void createMailboxShouldCreateAMailbox() throws Exception {
     }
 
     @Test
-    public void createMailboxShouldNotThrowIfMailboxAlreadyExist() throws Exception {
+    public void createMailboxShouldThrowIfMailboxAlreadyExist() throws Exception {
```

Comment:
s/Exist/Exists/
but why this change?

Reply:
Because now the JmxProbe forward an exception (feat @mbaechler ) hence the test was failing.